Assets/MRTK/SDK/Features/UX/Prefabs/Sliders/PinchSlider.prefab

```diff
@@ -1302,7 +1302,7 @@ Transform:
   m_PrefabAsset: {fileID: 0}
   m_GameObject: {fileID: 7307063430758334042}
   m_LocalRotation: {x: 0, y: 0, z: 0, w: 1}
-  m_LocalPosition: {x: 0, y: 0, z: 0.0125}
+  m_LocalPosition: {x: 0, y: 0, z: 0}
```

Comment:
Here is the behavior I am getting when I rotate the "SliderThumb" container within the prefab:

![PinchSliderFixBehavior](https://user-images.githubusercontent.com/53493796/105755581-5b387f00-5f00-11eb-9791-3a65e2865d03.gif)

The slider object snaps back to the slider axis after it is selected it again after rotating, is this expected?

Reply:
If you change the selection mode from Center to Pivot you end up rotating around the point (0, 0, 0) of the gameobject rather than the center of it's children, which I matches the experience described in the issue

![image](https://user-images.githubusercontent.com/39840334/105765968-27248480-5f27-11eb-8099-9dd19fb5c074.png)

Note that when using Center Mode, that the position of the gameobject itself changes as well as it's rotation.